.rubocop.yml

```diff
@@ -0,0 +1,37 @@
+require: rubocop-rspec
+
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+    - 'vendor/**/*'
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'app/assets/**/*'
+    - 'config/**/*'
+    - 'doc/**/*'
+    - 'spec/**/*'
+    - 'test/**/*'
+
+Rails:
+    Enabled: true
+
+Style/Encoding:
+  Enabled: true
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: single_quotes
+
+Style/Documentation:
+  Exclude:
+    - 'spec/**/*'
+    - 'app/**/*'
+
+Style/Next:
+  Enabled: false
```

Comment:
@hweng yeah, there are some examples in that issue I could see being annoying to have described as a "failure", since some of those are subjectively better syntax than using next.

Generally it seems like good practice to use next as a guard clause, though if it got annoying or we had use cases like the ones mentioned crop up, we could always disable again or experiment with the MinBodyLength setting? Just a thought.